### PR TITLE
Catchup for the past few days

### PR DESCRIPTION
**Pull request by Aleksa-Vig**
Catchup for the last few days

### CODE REVIEW

File: aleksa/containsDuplicate.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def containsDuplicate(self, numbers: List[int]) -> bool:
+        hashset = set()
+
+        #Time complexity: O(|numbers|)
+        #Memory complexity: O(|numbers|)
+
+        return any(number in hashset or hashset.add(number) for number in numbers)
```
Comment (zainaraza43) [inline aleksa/containsDuplicate.py:8]: HOLY BASED

File: aleksa/validPalindrome.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+class Solution:
+
+    def anum(self, c):
+        return (ord("A") <= ord(c) <= ord("Z")or ord("a") <= ord(c) <= ord("z")or ord("0") <= ord(c) <= ord("9"))
```
Comment (zainaraza43) [inline aleksa/validPalindrome.py:4]: interesting way to do it... RESPECT

**Review by zainaraza43 — APPROVED** [review]
LGTM :rocket:

File: aleksa/validPalindrome.py (hunk 2 above)
Comment (Aleksa-Vig) [reply aleksa/validPalindrome.py:4]: NO COMPILER TRICKS BABY

File: aleksa/containsDuplicate.py (hunk 1 above)
Comment (Aleksa-Vig) [reply aleksa/containsDuplicate.py:8]: one liner >:)

**Pull request merged**